### DISCUSSION TITLE
【声明】你们一群黑子（

### DISCUSSION THREAD

**Discussion by Soldier11-ObsidianBarracks** [share121/inter-knot#1711] [其他]
你们怎么尬黑我，绳网征兵就是抓绳匠进去处刑啊，居然还说我11号有对叛军和绳匠动用私刑的？全都是诽谤和尬黑
还有人说我和传奇~~绳斤~~绳匠法厄同有联系，笑死了，你们不会要说是random play的店长吧？他们也是这样和我说的，但如果他们是法厄同又怎么可能会和我一起打击叛军呢？

**Comment by Soldier11-ObsidianBarracks** [comment]
你们全是一群黑子尬黑我（确信

**Comment by Elysia-Eden-11** [comment]
哈哈哈哈
